recipe/meta.yaml

```diff
@@ -31,29 +30,34 @@ requirements:
     - wheel
     - llvmdev 11.1.*
     - llvm 11.1.*
```

Comment:
conda-forge tries t build v0.40.0 with `llmv` and `llvmdev` `14.0.*` https://github.com/conda-forge/llvmlite-feedstock/pull/69/files#diff-f3725a55bf339595bf865fec73bda8ac99f283b0810c205442021f29c06eea9aR31-R32. Should we consider to use this version? We already have it https://github.com/AnacondaRecipes/llvmdev-feedstock/blob/b7b0d5f14cc86126ec616c0626a7e7d3df215b9f/recipe/meta.yaml#LL1C1-L1C1

Reply:
see above, we can do it sure, is not that we should use the oldest supported though ?